src/components/SelectionList/BaseListItem.tsx

```diff
@@ -37,7 +38,7 @@ function BaseListItem<TItem extends ListItem>({
     const pressableRef = useRef<View | HTMLDivElement>(null);
 
     // Sync focus on an item
-    useSyncFocus(pressableRef, Boolean(isFocused));
+    useSyncFocus(pressableRef, Boolean(isFocused && !shouldPreventFocusSync));
```

Comment:
This name is a bit much to parse 😅 I suggest to remove the `prevent` from name, so that name stays positive - describing what it does, and not negated - which means I need to think more...

First we name it "prevent" but then we have to run `!` before it.

I suggest to use a simpler `shouldSyncFocus` name to use everywhere